source/manage/server-admin-gui.rst

```diff
@@ -24,19 +24,28 @@ Press *Apply changes* to store any changes.
 Metrics resolution
 ******************
 
-PMM collects metrics at three different time intervals.
-
+Metrics are collected at three intervals representing low, medium and high resolutions.
```

Comment:
You don't define what is meant by 'resolutions' - but we use the word a ton. I would encourage yo uto say "scraping intervals" or "frequency of data collection attempts" or something else to ensure the idea is communicated.

Reply:
Good spot. We've had some chats about this terminology already but no clear ... resolution. I'm concerned that the terms are not intuitive and I don't want the docs to keep having to explain terms because they are different to the UX. I will chat with our new UI guy and see if we can come up with a clearer explanation, or suggest a new UI metaphor.